Example/iOS-Example/Model.swift

```diff
@@ -11,32 +11,28 @@ import Foundation
 import FetchRequests
 
 class Model: NSObject {
-    typealias ObjectID = String
-    typealias RawData = [String: Any]
+    typealias ID = String
+    typealias RawData = CWJSON
 
     @objc dynamic
-    private(set) var objectID: ObjectID
+    private(set) var id: ID
 
     @objc dynamic
     private(set) var createdAt: Date = .distantPast
 
     @objc dynamic
     private(set) var updatedAt: Date = .distantPast
 
-    private var _data: RawData = [:]
-    @objc dynamic var data: RawData {
-        get {
-            return _data
-        }
-        set {
-            _data = newValue
+    @Observable
+    var data: RawData {
+        willSet {
```

Comment:
This is probably the most significant change we'd have to do:

* `data` is now @Observable, which inherently adds a didSet {} to post the change
* We now integrate the data in `willSet`, so `self.data` will be different from `integrate(data: data)`

Reply:
yeah hopefully that doesn't bring to much grief.